service/controller/resource/tcnpstatus/delete.go

```diff
@@ -0,0 +1,16 @@
+package tcnpstatus
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+)
+
+func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
+	err := r.ensure(ctx, obj)
```

Comment:
Same question as for the `instancesstatus` resource. Do we need to run this on delete events? I would like to prevent cluster deletion failures because of some stupid issues with status resources.

Reply:
I wasn't sure with this one - it might be good to keep this up to date for customers to see? Otherwise the CR status will stay static during deletion.

Not sure if that's important.

Reply:
I would say let's drop it. When somebody needs CR status updates for deleting TCs they can talk to us.